client/src/components/Card/GridCard.js

```diff
@@ -0,0 +1,157 @@
+import React from "react";
+import { Card, Col, Row } from "antd";
+import HelpBoard from "assets/how-does-work-images/Help-Board.gif";
+import LocalGlobal from "assets/how-does-work-images/Local&Global.gif";
+import RelevantContent from "assets/how-does-work-images/Relevant-content.gif";
+
+const GridCard = () => {
+     const { Meta } = Card;
+
+    return (
+        <div className="site-card-wrapper" style={{"marginTop": "-90px"}}>
+        <Row>
+          <Col span={8}>
+            <Card
+              cover={
+                <img
+                  alt="example"
+                  src={HelpBoard}
+                  style={{"marginLeft": "72px","marginRight": "883px", width: "326px", height:"290px","object-fit": "contain"}}
+                />
+              }
+              bordered={false}
+              title={"Help Board"}
+              headStyle={{width: "291px",
+                height: "28px",
+                "font-family": "Poppins",
+                "marginLeft": "107px",
+                "marginTop": "90px",
+                "font-size": "24px",
+                "font-weight": "600",
+                "font-stretch": "normal",
+                "font-style": "normal",
+                "line-height": "1.17",
+                "letter-spacing": "normal",
+                "color": "var(--color-primary-text)"}}
+            >
+              <Meta
+               style={
+                { "font-family": "WorkSans",
+                  "font-size":"20px",
+                  "width":"286px",
+                  "height":"90px",
+                  "text-align": "center",
+                  "marginLeft": "75px",
+                  "marginTop": "-39px",
+                  "marginRight": "886px",
```

Comment:
Lots of same comments as above, use theme colors, don't use pixels, use Rems, don't use inline styles, use styled-components, format this properly (applies to all files).

Reply:
same as previous comment for rem.have removed majority of inline styles

Reply:
done